airflow/emr_serverless/operators/emr.py

```diff
@@ -114,9 +114,10 @@ class EmrServerlessStartJobOperator(BaseOperator):
       Its value must be unique for each request.
     :param wait_for_completion: If true, waits for the job to start before returning. Defaults to True.
     :param aws_conn_id: AWS connection to use
+    :param config: extra configuration for the job.
     """
 
-    template_fields: Sequence[str] = ("application_id",)
+    template_fields: Sequence[str] = ("application_id", "execution_role_arn", "job_driver", "configuration_overrides")
```

Comment:
Why the additional template_fields here? I think I understand, but just to make sure.

Reply:
I was created a wrapper for the Emr serverless operator to be able to use templates on a project. Since I decided to open the PR to add kwargs to the hook I thought it was fair to add those templates and get rid of this workaround. Also, most of the aws airflow operators allow template usage on pretty much all parameters, so might aswell add them here